app/grandchallenge/cases/static/js/cases/ignore_enter_button.js

```diff
@@ -0,0 +1,5 @@
+$(document).keypress(function (e) {
+    if (e.which == 13) {
+        return false;
+    }
+});

```

Comment:
Why this method?

Reply:
The search field is nested in the form that has a submit button. Pressing enter after typing a search term (quite a common thing to do I think) will trigger the entire form to submit. So I thought it would make most sense to disable enter for the entire form. The search happens automatically when they type.